frontend/src/component/common/ApiTokenTable/ApiTokenTable.tsx

```diff
@@ -37,34 +19,27 @@ interface IApiTokenTableProps {
     columns: any[];
     rows: Row<object>[];
     prepareRow: (row: Row<object>) => void;
-    getTableProps: (
-        propGetter?: TablePropGetter<object> | undefined,
-    ) => TableProps;
-    getTableBodyProps: (
-        propGetter?: TableBodyPropGetter<object> | undefined,
-    ) => TableBodyProps;
     headerGroups: HeaderGroup<object>[];
     globalFilter: any;
 }
+
 export const ApiTokenTable = ({
     compact = false,
     setHiddenColumns,
     columns,
     loading,
     rows,
-    getTableProps,
-    getTableBodyProps,
     headerGroups,
     globalFilter,
     prepareRow,
 }: IApiTokenTableProps) => {
-    const isSmallScreen = useMediaQuery(theme.breakpoints.down('md'));
+    const isMediumScreen = useMediaQuery(theme.breakpoints.down('xl'));
```

Comment:
Should this really be `xl`? I'd expect `l` to be more natural?

Reply:
xl is still 1536px. With sidebar and on contemporary monitors this is how hiding columns turned out when testing screen width.

Reply:
Right. But shouldn't it be `isLargeScreen`, then? I'm just confused by the mismatch between the media query and the variable name. It looks like someone made a mistake somewhere.

Reply:
`isNotExtraLarge`?

Reply:
Sure? 🤷🏼